From 505fb5e41ac888590ae69cf503ebe9fbcdeaecd6 Mon Sep 17 00:00:00 2001
From: Steve deRosier <steve.derosier@lairdtech.com>
Date: Wed, 6 Jul 2016 16:41:26 -0700
Subject: [PATCH] laird: dts: wb50n: Fix mmc0 chip-detect pin

CD pin had incorrect active polarity. Fixed to be active low.

Bug: 9583
---
 arch/arm/boot/dts/wb50n.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/wb50n.dts b/arch/arm/boot/dts/wb50n.dts
index 8a39a8b43b1b..9fced057378d 100644
--- a/arch/arm/boot/dts/wb50n.dts
+++ b/arch/arm/boot/dts/wb50n.dts
@@ -65,10 +65,10 @@
 				pinctrl-names = "default";
 				pinctrl-0 = <&pinctrl_mmc0_clk_cmd_dat0 &pinctrl_mmc0_dat1_3 &pinctrl_mmc0_cd>;
 				status = "okay";
+				cd-gpios = <&pioC 26 GPIO_ACTIVE_LOW>;
 				slot@0 {
 					reg = <0>;
 					bus-width = <4>;
-					cd-gpios = <&pioC 26 GPIO_ACTIVE_HIGH>;
 				};
 			};
 
-- 
2.11.0

